migrator/clone/rocksdb/db_clone_manager_impl.go

```diff
@@ -3,6 +3,7 @@ package rocksdb
 import (
```

Comment:
What are the extra stuff added?

So while we upgrade to Postgres, if we keep previous valid which could be used for rollback and a fake current which could be at the version right before 73 DB, it looks like a valid clone set to old versions.

Reply:
Answering this in the other question.